net/socks5/init.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
don't name this file `init.go`... that's too similar to the Go magic `func init()` for comfort.